src/main.cpp

```diff
@@ -26,6 +26,8 @@ namespace YimMenu
 
 		while (g_Running)
 		{
+			if (GetAsyncKeyState(VK_F12))
+				g_Running = false;
```

Comment:
There's already an unload key present in the code base as VK_DELETE in the `src/game/hooks/GUI/Window.cpp` file.